addons/mail/models/mail_thread.py

```diff
@@ -2020,7 +2020,7 @@ def _notify_thread(self, message, msg_vals=False, model_description=False, mail_
             if inbox_pids:
                 message_values = message.message_format()[0]
                 for partner in self.env['res.partner'].browse(inbox_pids):
-                    notifications.append([(self._cr.dbname, 'ir.needaction', partner), dict(message_values)])
+                    notifications.append([(self._cr.dbname, 'ir.needaction', partner.id), dict(message_values)])
```

Comment:
Actually it seems browsing partners is not necessary, you could do it directly with inbox_pids, isn't it ?

Reply:
actually yes, changes done